rust-src/concordium_base/benches/eddsa_benchmarks.rs

```diff
@@ -32,13 +28,10 @@ pub fn bench_sign(c: &mut Criterion) {
     }
     let d = a.clone();
 
-    let sk = SecretKey::generate(&mut csprng);
-    let pk = PublicKey::from(&sk);
-    let expanded_sk = ExpandedSecretKey::from(&sk);
-    let sig = expanded_sk.sign(a.as_slice(), &pk);
-    c.bench_function("sign {}", move |b| {
-        b.iter(|| expanded_sk.sign(a.as_slice(), &pk))
-    });
+    let signing = SigningKey::generate(&mut csprng);
```

Comment:
minor comment: I would call this something else like signing_key or sk

Reply:
Fixed.